diff --git a/data/templates/default.latex b/data/templates/wg21.latex
index 7d539fe..723e93e 100644
--- a/data/templates/default.latex
+++ b/data/templates/wg21.latex
@@ -406,8 +406,27 @@ $if(colorlinks)$
 $else$
   hidelinks,
 $endif$
-  pdfcreator={LaTeX via pandoc}}
+  pdfcreator={mpark/wg21}}
 
+\makeatletter
+\def\@maketitle{
+  \newpage \null \vskip 2em
+  {\center \LARGE \@title \par}
+  \vskip 1.5em
+  \begin{flushright}
+    \begin{tabular}{ll}
+Document \#:&$document$\\
+Date:       &\@date\\
+Project:    &Programming Language C++\\
+Audience:   $for(audience)$&$audience$\\$endfor$
+$if(revises)$
+Revises:    &$revises$\\
+$endif$
+Reply-to:   \@author
+    \end{tabular}
+  \end{flushright}
+}
+\makeatother
 $if(title)$
 \title{$title$$if(thanks)$\thanks{$thanks$}$endif$}
 $endif$
@@ -423,7 +442,7 @@ $else$
 $endif$
 \subtitle{$subtitle$}
 $endif$
-\author{$for(author)$$author$$sep$ \and $endfor$}
+\author{$for(author)$&$author.name$\\$for(author.email)$&<$author.email$>\\$endfor$$endfor$}
 \date{$date$}
 $if(beamer)$
 $if(institute)$
